### PR TITLE
Release-20161019 hotfix news post

### PR DESCRIPTION
**Pull request by pchote**
website

gamenews

### CODE REVIEW

**Comment by pchote** [comment]
I have noticed and fixed the typo in the game news (second 20161015 should be 20161019).

File: content/gamenews [hunk 1]
```diff
@@ -1,4 +1,4 @@
-2016-10-15 20-00-00:
-	Title: A new stable release is available!
-	Author: Oliver Brakmann
-	Content: We are happy to announce a new stable release of OpenRA!\n\nA new content installer, stability improvements, especially for servers, new missions and many more interesting changes and fixes await you.\n\nHead over to our website at http://openra.net to see the full announcement, and take a look at the full changelog at http://changelog.openra.net to get an impression of all the changes.\n\nWe wish you a lot of fun with this new release!
+2016-10-19 20-00-00:
+	Title: Release 20161019 hotfix now available!
+	Author: Paul Chote
+	Content: LANDCOM 16 HQS.\nTOP SECRET\nTO: ALL FIELD COMMANDERS\n\nA SURPRISE AMBUSH OF MULTIPLAYER ISSUES\nHAS LEFT THE 20161015 RELEASE CRITICALLY\nWEAKENED.  REPORT TO www.openra.net FOR A FULL CASUALTY REPORT.\n\nPROCEEED TO www.openra.net/download/ FOR THE\nREINFORCED 20161019 RELEASE THEN CONTINUE\nTO THE MULTIPLAYER LOBBY AND DESTROY ALL\nENEMY OPPOSITION.\n\nTRANSMISSION ENDS.
```
Comment (abcdefg30) [inline content/gamenews]: I think it would look a small bit better if you added another new line at the end.

Comment (pchote) [inline content/gamenews]: That then forces it to scroll, which we try to avoid.

Comment (pchote) [inline content/gamenews]: We could perhaps remove the "LANDCOM 16 HQS." line if you think the bottom margin is important.

File: content/news/release-20161019.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+PROCEEED TO <a href="/download/">www.openra.net/download/</a> TO OBTAIN THE FORTIFIED RELEASE.
+RENDEZVOUS WITH OTHER COMMANDERS IN THE MULTIPLAYER LOBBY AND DESTROY
+ALL ENEMY OPPOSITION.
+
+TRANSMISSION ENDS.
+</pre>
+</div>
+
+A release-20161019 hotfix is now available to fix two critical multi-player bugs that were missed during the playtesting phase. The original release-20161015 release announcement is included below:
```
Comment (obrakmann) [inline content/news/release-20161019.md]: "The original announcement for release-20161015 is included below" flows a bit better, I think.

File: content/news/release-20161019.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Other new features and fixes include the following:
+
+* Fixes to a lot of crashes.
+* Fixes to a number of bugs that impacted game servers, such as the infamous "grey ping bug".
+* New configuration options for customizing mouse scrolling behaviour.
+* New hotkeys for navigating to map edges, and bookmarking positions on the map.
+* New hotkeys for controlling the music player, as well as producing five units at once.
+* Improved aircraft reloading behaviour.
+
+There are many more changes than we are able to mention here, including the usual round of balance changes. Head on over to the [Changelog page on our wiki](https://github.com/OpenRA/OpenRA/wiki/Changelog/561383b9ba791a557cde37616f42a8df7d8a3182) to see the complete list.
```
Comment (obrakmann) [inline content/news/release-20161019.md]: Oh, you might want to link to the updated changelog as well: https://github.com/OpenRA/OpenRA/wiki/Changelog/0ae64ef1b6dded0a55900ab67561c2d56885c461

**Comment by pchote** [comment]
Updated with suggestions and the new release id.

**Pull request merged**